extractor/fulltext/launch_single.py

```diff
@@ -0,0 +1,19 @@
+import os
+import sys
+# sys.path.append(".")
+from fulltext import logger
+from fulltext import convert
+
+log = logger.getLogger('fulltext')
+
+if __name__ == '__main__':
+    if len(sys.argv) <= 1:
+        sys.exit('No file path specified')
+    path = sys.argv[1].strip()
+    try:
+        sys.stdout.write('Path: %s\n' % path)
```

Comment:
Should these lines also go to the logger?

Reply:
Yep, good catch.